assets/queries/cloudFormation/ec2_subnet_unable_connect_cluster/test/positive.yaml

```diff
@@ -0,0 +1,14 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: A sample template
+Resources:
+   mySubnet:
+    Type: AWS::EC2::Subnet
+    Properties:
+      MapPublicIpOnLaunch: true
+      VpcId:
+        Ref: myVPC
```

Comment:
To make this sample valid either remove `Ref:` and leave it like this `VpcId: myVPC` or add the parameter myVPC of type string to the top of the file.